### DISCUSSION TITLE
'Pick from not-recently played' random selection algorithm

### DISCUSSION THREAD

**Discussion by smileyhead** [ppy/osu#13063] [Ideas]
**Describe the new feature:** The game would store the date on which a map was last played, and this algorithm would use that date to mostly pick from maps you haven't played in a while.
Could also be a mode for #5757.

**Comment by bdach** [comment]
Well, except for the fact that sort of selection algorithm is *not random*, could be interesting.

**Comment by smileyhead** [comment]
You're right, I did word it wrong.
I didn't mean it would always pick the one map you haven't played for the longest time, but that it would pick randomly from the ones you haven't played in a while.

**Comment by peppy** [comment]
This would definitely be nice to have. Can hopefully be implemented without any config storage requirements (just ordering by last played and falling back to a random method).
